syft/serde.py

```diff
@@ -157,6 +155,74 @@ def deserialize(
     return _detail(worker, simple_objects)
 
 
+def serialize_tensor(tensor) -> bin:
+    """Serialize the tensor using as default Torch serialization strategy
+    This function can be overridden to provide different tensor serialization strategies
+
+    Args
+        (torch.Tensor): an input tensor to be serialized
+
+    Returns
+        A serialized version of the input tensor
+
+    """
+    return torch_tensor_serializer(tensor)
+
+
+def deserialize_tensor(tensor_bin) -> torch.Tensor:
+    """Deserialize the input tensor passed as parameter into a Torch tensor.
+    This function can be overridden to provide different deserialization strategies
+
+    Args
+        tensor_bin: A binary representation of a tensor
+
+    Returns
+        a Torch tensor
+    """
+    return torch_tensor_deserializer(tensor_bin)
+
+
+def numpy_tensor_serializer(tensor: torch.Tensor) -> bin:
+    """Strategy to serialize a tensor using numpy npy format.
+    If tensor requires to calculate gradients, it will detached.
+    """
+    if tensor.requires_grad:
+        print(
```

Comment:
Rather than printing this, I think we should raise it as a warning ie warnings.warn